diesel/src/sqlite/types/json.rs

```diff
@@ -0,0 +1,1073 @@
+//! Support for JSON and JSONB values under PostgreSQL.
```

Comment:
nit - Should this comment be: `//! Support for JSON and JSONB values under SQLite.`?

Great work! Hoping this lands on the next diesel version :)

Reply:
> Can you submit a PR to fix that?

Done: #4307